### DISCUSSION TITLE
Play on cell phone

### DISCUSSION THREAD

**Discussion by rpeveri** [EmulatorJS/EmulatorJS#338] [Ideas]
This project is spectacular and works very well. The only drawback I find is that you can't play it on your cell phone. I hope that soon it will be playable on all devices.
Greetings and thanks for the effort to carry out the project.

**Comment by ethanaobrien** [comment]
Actually, it's already playable with every mobile/desktop that supports it. If a phone doesn't support it it's either because it's too outdated or because an api is needed that isn't supported in the mobile browser. Sorry about that

**Reply by Vivek230800** [reply]
Could you please share me a readme file about how to set this up? I am having problems in setting this up.

**Reply by Ramaerel** [reply]
Hi @Vivek230800 ,

If you're referring to EmulatorJS as a whole I would (personally) recommend you check out my project here: https://github.com/Ramaerel/emulatorjs-GameLibrary

It's almost entirely drag and drop with almost all web hosts, and only requires something along the lines of XAMPP for Windows if your self hosting. You just need to download it and this EmulatorJS project. It isn't the prettiest, but if you can't figure out how to set up EmulatorJS on your own site it simplifies it enough for personal use 🙂

(Hopefully this is OK to plug here, if not I can delete it)

**Reply by ethanaobrien** [reply]
It's all good. I do also have your project in the readme in case you didn't know

**Reply by Ramaerel** [reply]
I actually didn't. Awesome, thank you 😁

**Reply by Vivek230800** [reply]
Thank you @Ramaerel . Appreciate your help. 🙏👍

**Comment by Ramaerel** [comment]
I forgot to make an issue about this but one of the latest updates broke compatibility with iOS devices, if that's what the post is talking about. Unfortunately I can't test further as I don't actually own one, just had access to one for a few days

**Reply by ethanaobrien** [reply]
Could you open an issue for this since you technically found it? I found that it doesn't work (as you said) and I'm pretty sure it's a syntax thing. Not sure when I can debug this
